plugins/meta/tuning/README.md

```diff
@@ -2,11 +2,11 @@
 
 ## Overview
 
-This plugin can change some system controls (sysctls) in the network namespace.
+This plugin can change some system controls (sysctls) and several netlink attributes (promiscuous mode and mac) in the network namespace.
```

Comment:
I would change this to "several interface attributes (promiscuous mode, MTU, and MAC address)"

Reply:
Thanks. Changed.